src/Html2Text.php

```diff
@@ -30,21 +47,38 @@ class Html2Text {
 	 * </ul>
 	 *
 	 * @param string html the input HTML
+	 * @param array An array of options of the Html2Text::OPT_* variety
 	 * @return string the HTML converted, as best as possible, to text
 	 * @throws Html2TextException if the HTML could not be loaded as a {@link DOMDocument}
 	 */
-	static function convert($html) {
+	static function convert($html,$options=array()) {
+
+		// reset
+		Html2Text::$_indexedUrls = array();
+
+		// DOMDocument doesn't support empty value and throws an error
+		if (!$html) {
+			return '';
+		}
+
 		// replace &nbsp; with spaces
 		$html = str_replace("&nbsp;", " ", $html);
 
 		$html = static::fixNewlines($html);
 
 		$doc = new \DOMDocument();
-		if (!$doc->loadHTML($html)) {
+		$doc->strictErrorChecking = FALSE;
+		$doc->recover = TRUE;
+		$doc->xmlStandalone = true;
```

Comment:
It would be nicer to add a comment for each of the above 3 lines. Telling the reason of the usage.